models/unet-2d-nuclei-broad/UNet2DNucleiBroad.model.yaml

```diff
@@ -41,39 +41,37 @@ outputs:
 
 prediction:
     preprocess:
-        - spec: https://github.com/bioimage-io/pytorchbioimageio:configurations/transformations/NormalizeZeroMeanUnitVariance.transformation.yaml:9fec4dd
-          kwargs:
-              eps: 1.0e-6  # currently all kwargs need to be specified (here or in the consumer)
-              apply_to: [0]
+        - spec: https://raw.githubusercontent.com/FynnBe/pytorch-bioimage-io/pybio/specs/transformations/NormalizeZeroMeanUnitVariance.transformation.yaml
+          kwargs: {apply_to: [0]}
     weights:
-        source: TODO
+        source: doi:TODOzenodo
         hash: {md5: TODO}
     postprocess: null
     dependencies: conda:../environment.yaml
 
 training:
     setup:
         reader:
-            spec: https://github.com/bioimage-io/pytorchbioimageio:configurations/readers/BroadNucleusData.reader.yaml:9fec4dd
+            spec: https://raw.githubusercontent.com/bioimage-io/python-bioimage-io/master/specs/readers/BroadNucleusData.reader.yaml
         sampler:
-            spec: https://github.com/bioimage-io/pytorchbioimageio:configurations/samplers/GrayscaleImageSampler.sampler.yaml:9fec4dd
+            spec: https://raw.githubusercontent.com/FynnBe/pytorch-bioimage-io/pybio/specs/samplers/GrayscaleImageSampler.sampler.yaml
         preprocess:
-            - spec: https://github.com/bioimage-io/pytorchbioimageio:configurations/transformations/NormalizeZeroMeanUnitVariance.transformation.yaml:9fec4dd
-              kwargs: {eps: 1.0e-6, apply_to: [0]}
+            - spec: https://raw.githubusercontent.com/FynnBe/pytorch-bioimage-io/pybio/specs/transformations/NormalizeZeroMeanUnitVariance.transformation.yaml
+              kwargs: {apply_to: [0]}
         loss:
-            - spec: https://github.com/bioimage-io/pytorchbioimageio:configurations/transformations/Sigmoid.transformation.yaml:9fec4dd
+            - spec: https://raw.githubusercontent.com/FynnBe/pytorch-bioimage-io/pybio/specs/transformations/Sigmoid.transformation.yaml
               kwargs: {apply_to: [0]}
-            - spec: https://github.com/bioimage-io/pytorchbioimageio:configurations/transformations/BCELoss.transformation.yaml:9fec4dd
+            - spec: https://raw.githubusercontent.com/FynnBe/pytorch-bioimage-io/pybio/specs/transformations/BCELoss.transformation.yaml
         optimizer:
             source: torch.optim.Adam
-            kwargs: {lr: 0.002}
+            default_kwargs: {lr: 0.002}
```

Comment:
I think it should just be kwargs here, but I see that this is a bit ambiguous, because we don't have a special
spec for the optimizer (yet?). But it's not really a part of the specification for this model.

Reply:
On second thought I guess this is ok: `source` always goes with `required_kwargs` and / or `default_kwargs` (which I would rather call `optional, see above`).